src/wp-includes/default-filters.php

```diff
@@ -590,7 +590,6 @@
 add_action( 'enqueue_block_editor_assets', 'wp_enqueue_editor_format_library_assets' );
 add_action( 'enqueue_block_editor_assets', 'wp_enqueue_global_styles_css_custom_properties' );
 add_filter( 'wp_print_scripts', 'wp_just_in_time_script_localization' );
-add_action( 'wp_print_scripts', 'wp_print_delayed_inline_script_loader' );
```

Comment:
Removed since now printed in `WP_Scripts::do_items()`. The move is necessary since we need to rely on `$wp_scripts->to_do` which is only available after `$wp_scripts->all_deps()` has been called.